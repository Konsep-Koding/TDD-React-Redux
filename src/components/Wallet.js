import React, { Component } from 'react'

export default class Wallet extends Component {
    render() {
        return (
          <div>Helo</div>
        )
    }
}
