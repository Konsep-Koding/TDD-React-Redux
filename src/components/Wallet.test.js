import React from 'react';
import { shallow, configure } from 'enzyme';
import Wallet from './Wallet';
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter(), disableLifecycleMethods: false });

describe('Wallet', () => {
    const wallet = shallow(<Wallet/>)
    it('Render', () => {
        expect(wallet).toMatchSnapshot();
});
});