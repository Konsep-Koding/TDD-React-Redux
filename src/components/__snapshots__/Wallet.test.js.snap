// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Wallet Render 1`] = `ShallowWrapper {}`;
